rd_ui/app/views/dashboard.html

```diff
@@ -42,7 +42,7 @@ <h3 class="panel-title">
                     </h3>
                 </div>
 
-                <visualization-renderer visualization="widget.visualization" query-result="queryResult"></visualization-renderer class="panel-body">
+                <visualization-renderer visualization="widget.visualization" query-result="queryResult" hide-filters="dashboard.dashboard_filters_enabled"></visualization-renderer class="panel-body">
```

Comment:
Fair enough. I expected that it wouldn't be necessary to know which queries are filtered or not, but making it clear is a good idea. How about if the widget still indicates the filter state of itself without including all the bulky filter UI repeated over and over?